.github/workflows/daily.yml

```diff
@@ -31,8 +38,13 @@ jobs:
           - ruby:2.7 node:14
           - ruby:3 node:16
           - ruby:3 node:14
-          - openjdk:8-jdk@https://raw.githubusercontent.com/docker-library/openjdk/master/8/jdk/buster/Dockerfile node:14 python:3.7 --override-from buildpack-deps:buster
-          - openjdk:11@https://raw.githubusercontent.com/docker-library/openjdk/master/11/jdk/buster/Dockerfile node:14
+        platform:
+          - linux/amd64,linux/arm/v7,linux/arm64/v8
```

Comment:
How come we are using one value here and not 3 different ones?

Reply:
I think using one platform per run might also solve your issue with forcing an update. Then it will be easier to check if just one set of images has changed. It will also make it easier to see what's happening in GitHub Actions.

Reply:
Oh. I would have to read up on it. Not sure either. I hope we can split because that would be much faster and probably help keep the code simpler.

Reply:
Here is some documentation that I just found: https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

Basically it would require building all of the images and then have a 2nd pipeline run once the first is complete to combine them all together. I am totally game to do that, but it seems like way more work for not a lot of gain.

Spit balling on options though, the easiest way would be to create a new workflow for each combo. To keep things sane, we could keep a "master combo list" and have a workflow that generates the sub workflows based on the master list.

Let me know how you would want to proceed.

Reply:
What about something like the following where the platform is not part of the matrix and we have 3 parallel jobs and manifest job that depends on them?

Something like:

```
jobs:
  x86:
    # ....
  arm7:
    # ...
  arm8:
    # ....
   manifest:
     needs: [x86, arm7, arm8]
```

We can get rid of the two OpenJDK special cases, or maybe adding some complex `if` in there to make it still work.

If it doesn't work, generating the workflow file would be a good option too.

I think separate job for each platform would still be simpler as we won't have to deal with any corner cases of update checking and it will run in parallel so not over 4 hours. Do you think that would be too much work too or worth the effort?

Reply:
If we split it up by architecture then every image would have to complete before the manifests could start being created. I think auto generating the jobs from a master file would be the way to go.

I will see if I can throw something together.

Reply:
Yeah the `needs` part of my example above should take care of that. The manifest job will wait for all the platform jobs to finish running in parallel.